src/style.scss

```diff
@@ -1,3 +1,4 @@
+.react-cm-ui {
```

Comment:
Let's maybe thing about this name.
We might want like `.hc` or `.healthy-church` or `.hc-ui` or `.healthy-church-ui`.

Thoughts?

cc
@JLeeC
@sircris
@War777
@KacosPro
@razmik-medoxi
@absqueued

Reply:
I wonder why we wrapped modular styles with a class ?
@IlyaRadinsky can you please explain what are the benefits of doing it and why we couldn't do it in another way ?

Reply:
@razmik-medoxi This is in service of **[Product Backlog Item 97569: [Tech Debt] - Healthy Church Widgets Framework - Ensure CSS shipped with widgets CANNOT affect anything on consuming page, outside of widget's container](https://dev.azure.com/saddlebackchurch/Church%20Management/_workitems/edit/97569)**.  So, for example, let's say React CM UI styles `<h1>` elements and other really common elements.  When we export CSS from React CM UI and bundle it as part of a widget intended for use on an external website (e.g. saddleback.com), that style will "spill over" from the widget and affect the consuming website, which is highly undesirable.  So this is designed to prevent that.

See for example:
https://hc-example-website.azurewebsites.net/ (no widgets)
https://hc-example-website.azurewebsites.net/Form (has the Forms widget on it)

Compare for example the top navigation bar (text gets bolder and such).
This is due to widget CSS (which includes React CM UI CSS).  That's what we're trying to prevent here.

Reply:
Fair enough, I thought we encapsulated the widgets but it seems the styling didn't work well with that solution

Reply:
`.hc-ui ` is the number 1 candidate for me =))